pkgs/development/python-modules/hyperpyyaml/default.nix

```diff
@@ -0,0 +1,39 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pytestCheckHook
+, pyyaml
+, ruamel-yaml
+}:
+
+buildPythonPackage rec {
+  pname = "hyperpyyaml";
+  version = "1.2.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "speechbrain";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-tC4kLJAY9MVgjWwU2Qu0rPCVDw7CjKVIciRZgYhnR9I=";
+  };
+
+  propagatedBuildInputs = [
+    pyyaml
+    ruamel-yaml
```

Comment:
We need two yaml libraries?

Reply:
Yes, it relies on both of them: https://github.com/speechbrain/HyperPyYAML/blob/1855d6172c3866a3c2c2e87e32ade0cd79abeff3/setup.py#L21